### PR TITLE
Fix example output in documentation

### PR DESCRIPTION
**Pull request by AVHopp**
This PR fixes a bug that erased the output of the examples in the documentation.

The package `jupytext` which was introduced earlier for the conversion of the examples to jupyter notebooks does not include the output of executed notebooks to a converted markdown **by design**. This PR thus re-introduces `jupyter` for this step of the conversion.

Also, this PR introduces some visual fixes to all of the examples, since the comments were not rendered properly.
**IMPORTANT:** The only changes in the examples are additions and deletions of new empty lines. No other changes were done in the examples, it is thus fully sufficient to locally build the docs and review the examples there.

### CODE REVIEW

**Review by Scienfitz — APPROVED** [review]
can you add screenshots of the situation before / after? otherwise its hard to even understand why the newline changes were needed

**Comment by AVHopp** [comment]
For comparison: Here are some screenshots from how examples `main` currently look like:

- Comments not being formatted properly:
image
- Executed code not in the file:
image

And this how how they should look like and do on this branch:
- Comments are being formatted properly (this is another example, but you get the point): 
image

- Output of code execution in file: 
image

**Review by AdrianSosic — APPROVED** [review]
Hi @AVHopp, I think you have a much better overview here anyway, so I'll accept to unblock you.

Just two small questions/comments:
* I think the printed output of the **particular** example you showed (i.e. the progess bars) is something that I would rather not want to see in the docs, but I guess this is the price we currently have to pay if we want the see the output of other calls...
* Just to understand that right: in the previous `p2j` approach, the added newlines were NOT necessary, but this had the downside that newlines where automatically added everywhere, correct? Also, just want to understand the implications: we now always must keep in mind to have a blank line in between, since otherwise things will silently fail, right? ... Not ideal, but I guess there is no alternative? Or can we somehow control it via a flag to `nbconvert` or similar?

**Comment by AVHopp** [comment]
> Hi @AVHopp, I think you have a much better overview here anyway, so I'll accept to unblock you.
> 
> Just two small questions/comments:
> 
> * I think the printed output of the **particular** example you showed (i.e. the progess bars) is something that I would rather not want to see in the docs, but I guess this is the price we currently have to pay if we want the see the output of other calls...
> * Just to understand that right: in the previous `p2j` approach, the added newlines were NOT necessary, but this had the downside that newlines where automatically added everywhere, correct? Also, just want to understand the implications: we now always must keep in mind to have a blank line in between, since otherwise things will silently fail, right? ... Not ideal, but I guess there is no alternative? Or can we somehow control it via a flag to `nbconvert` or similar?

Regarding these two points:

1.  There is still a lot to do, but I would leave that for another PR. It is in principle possible to get rid of specific lines: At the end, I need to perform a clean-up of all the file in general by going through all of the lines and remove some. This currently happens for removing the plots in backtesting. Hence, removing the lines with the progress can also be done there, but I would do that in another PR.
2. This is my interpretation. I did not look into these things in detail, since it worked :D I do not know whether we have an alternative here.

If you want more information on any of this, feel free to create a comment for blocking this PR. If not, I'll rebase and merge today.

**Pull request merged**
